atm.js

```diff
@@ -1,3 +1,150 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+ //DEPOSIT MONEY FROM CHECKING
+
+
+
+  var checkingDeposit = $("#checking .deposit")
+  var savingsDeposit = $(“#savings .deposit”)
+
+
+  checkingDeposit.on("click", function(){
+
+
+    var checkingUserInput = $("#checking .input").val()
+    console.log("This is the user input:" + checkingUserInput)
+
+     create a variable that stores "checking"‘s balance(in this exercise = $0)
+    var checkingOutputBalance = $("#checking .balance")
+
+
+    checkingBalance += parseInt(checkingUserInput)
+    console.log("This is the checking balance " + checkingBalance)
+
+     display "$" + "checkingBalance" ($0, or whatever user inputs in our balance ("outputBalance"))
+    checkingOutputBalance.text("$" + checkingBalance);
+  })
+
+
+
+
+
+ var checkingDeposit = $(".withdraw")
+
+
+ checkingDeposit.on("click", function() {
+
+
+ var checkingUserInput = $("#checking .input").val()
+ console.log("This is the user input:" + checkingUserInput);
+
+
+ var checkingOutputBalance = $("#checking .balance")
+
+
+ checkingBalance -= parseInt(checkingUserInput)
+ console.log("This is the checking balance " + checkingBalance);
+
+ checkingOutputBalance.text("$" + checkingBalance);
+
+
+
+
+ var savingsDeposit = $("#savings .deposit")
+
+ savingsDeposit.on("click", function() {
```

Comment:
Please indent your code properly so it's easy to tell, for example, what code is inside of this click listener's callback function.